tools/builder/src/scs_translator.cpp

```diff
@@ -73,7 +73,7 @@ bool SCsTranslator::processString(const String &data)
 
 #if defined( __WIN32__ ) || defined( _WIN32 )
   input = antlr3StringStreamNew((pANTLR3_UINT8)data.c_str(), ANTLR3_ENC_UTF8, (ANTLR3_UINT32)data.length(), (pANTLR3_UINT8)"scs");
-#elif defined( __APPLE_CC__)
+#elif defined( __APPLE_CC__) || (defined( __linux__ ) && defined(ANTLR3C_NEW_VERSION))
```

Comment:
Newer version (3.4) has `antlr3StringStreamNew()` instead of `antlr3NewAsciiStringCopyStream()`